docs/plugins/dffml_model.rst

```diff
@@ -287,26 +352,590 @@ Linear Regression Model implemented using scikit. Models are saved under the
+
+  - default: 1e-08
+
+- n_iter_no_change: Integer
+
+  - default: 10
+
+scikitqda
+~~~~~~~~~
+
+*Core*
+
+No description
+
+**Args**
+
+- directory: String
+
+  - default: /home/user/.cache/dffml/scikit-scikitlr
+  - Directory where state should be saved
+
+- predict: String
+
+  - Label or the value to be predicted
+
+- priors: BaseConfigurable.parser_helper
+
+  - default: None
+
+- reg_param: float
+
+  - default: 0.0
+
+- store_covariance: BaseConfigurable.type_for.<locals>.<lambda>
+
+  - default: False
+
+- tol: float
+
+  - default: 0.0001
+
+scikitrfc
+~~~~~~~~~
+
+*Core*
+
+No description
+
 **Args**
 
 - directory: String
 
-  - default: /home/user/.cache/dffml/scikit
+  - default: /home/user/.cache/dffml/scikit-scikitlr
   - Directory where state should be saved
 
 - predict: String
 
-  - Label or the value to be predicted
\ No newline at end of file
+  - Label or the value to be predicted
+
+- n_estimators: String
+
+  - default: warn
+
+- criterion: String
+
+  - default: gini
+
+- max_depth: BaseConfigurable.parser_helper
+
+  - default: None
+
+- min_samples_split: Integer
+
+  - default: 2
+
+- min_samples_leaf: Integer
+
+  - default: 1
+
+- min_weight_fraction_leaf: float
+
+  - default: 0.0
+
+- max_features: String
+
+  - default: auto
+
+- max_leaf_nodes: BaseConfigurable.parser_helper
+
+  - default: None
+
+- min_impurity_decrease: float
+
+  - default: 0.0
+
+- min_impurity_split: BaseConfigurable.parser_helper
+
+  - default: None
+
+- bootstrap: BaseConfigurable.type_for.<locals>.<lambda>
+
+  - default: True
+
+- oob_score: BaseConfigurable.type_for.<locals>.<lambda>
+
+  - default: False
+
+- n_jobs: BaseConfigurable.parser_helper
+
+  - default: None
+
+- random_state: BaseConfigurable.parser_helper
+
+  - default: None
+
+- verbose: Integer
+
+  - default: 0
+
+- warm_start: BaseConfigurable.type_for.<locals>.<lambda>
+
+  - default: False
+
+- class_weight: BaseConfigurable.parser_helper
+
+  - default: None
+
+scikitsvc
+~~~~~~~~~
+
+*Core*
+
+No description
+
+**Args**
+
+- directory: String
+
+  - default: /home/user/.cache/dffml/scikit-scikitlr
+  - Directory where state should be saved
+
+- predict: String
+
+  - Label or the value to be predicted
+
+- C: float
+
+  - default: 1.0
+
+- kernel: String
+
+  - default: rbf
+
+- degree: Integer
+
+  - default: 3
+
+- gamma: String
+
+  - default: auto_deprecated
+
+- coef0: float
+
+  - default: 0.0
+
+- shrinking: BaseConfigurable.type_for.<locals>.<lambda>
+
+  - default: True
+
+- probability: BaseConfigurable.type_for.<locals>.<lambda>
+
+  - default: False
+
+- tol: float
+
+  - default: 0.001
+
+- cache_size: Integer
+
+  - default: 200
+
+- class_weight: BaseConfigurable.parser_helper
+
+  - default: None
+
+- verbose: BaseConfigurable.type_for.<locals>.<lambda>
+
+  - default: False
+
+- max_iter: Integer
+
+  - default: -1
+
+- decision_function_shape: String
+
+  - default: ovr
+
+- random_state: BaseConfigurable.parser_helper
+
+  - default: None
```

Comment:
Why does this exist?

Reply:
Well, docs.py needs to generate the args for everything. Those definitions are weird https://github.com/intel/dffml/issues/194. We could make it so that the docs script has a list of things it shouldn't print out args() for. That would be good.

Reply:
It was working in 3ed5298, we can change the condition to a list maybe here https://github.com/intel/dffml/pull/190/commits/3ed529811d2a3a1558365b2dd52e9a00c195b853#diff-69087563aad9b14661594d9bd805fcd2R173-R177

Reply:
Ah, yes, I didn't remember to change the if statement as well. Maybe we could add

```python
"\n\n".join(docs) if module_name not in list_of_module_names_not_to_include
```

and load `list_of_module_names_not_to_include` from a file, similarly to how we load from `scripts/care`. Or you could make it so we use [configparser](https://docs.python.org/3/library/configparser.html) to read the docs.py config from a file and put the list in there.

Reply:
so, consolidate the scripts/care and the list of modules not to write args for into one config file that gets read with configparser.